Tests/Remora.Discord.Gateway.Tests/Tests/DiscordGatewayClientTests.cs

```diff
@@ -101,6 +103,7 @@ public async Task CanConnectAsync()
         var transportMockDescriptor = ServiceDescriptor.Singleton(typeof(IPayloadTransportService), transportMock);
 
         var services = new ServiceCollection()
+            .Configure<ResponderDispatchOptions>(() => new())
```

Comment:
You should do this in `AddDiscordGateway` instead; as it is, any consumer will also have to do this which is quite unintuitive.